pkg/controller/pkg/revision/hook.go

```diff
@@ -121,13 +119,13 @@ func NewConfigurationHooks() *ConfigurationHooks {
 
 // Pre sets status fields based on the configuration package.
 func (h *ConfigurationHooks) Pre(ctx context.Context, pkg runtime.Object, pr v1alpha1.PackageRevision) error {
-	pkgConfig, ok := pkg.(*pkgmeta.Configuration)
+	_, ok := pkg.(*pkgmeta.Configuration)
```

Comment:
Should we just make this a complete no-op method for now?

Reply:
My reasoning for keeping the check here is that I think we will need it in the short term future and if we did a no-op here then it would match `NopHooks`, and then it would strictly be duplicate code and in that case it seems like we should just remove `ConfigurationHooks` entirely for now. Since we would be reintroducing this soon, the lowest churn path seemed like to keep it around with minimal functionality, then fill it in with more as necessary.